charts/datasaur/templates/config/intelligence-env-configMap.yaml

```diff
@@ -2,8 +2,21 @@ apiVersion: v1
 kind: ConfigMap
 metadata:
   name: intelligence-env
-data: 
+data:
   NLTK_PROVIDER_CORENLP_URL: http://{{ .Release.Name }}-core-nlp-service:9000
-  NLTK_PROVIDER_PORT: '9090'
+  NLTK_PROVIDER_PORT: "9090"
   SPARKNLP_JSL_HOST: http://{{ .Release.Name }}-jsl-server-service:5000
-  SPARKNLP_PORT: '9091'
\ No newline at end of file
+  SPARKNLP_PORT: "9091"
+
+  # Datasaur Dinamic environment variables
+  DATASAUR_ASSIST_VULCAN_BASIC_AUTH: {{ .Values.intelligence.datasaurDinamic.authKey }}
```

Comment:
this is too, please give the instructions in values.sample.yaml in our private repo for scheme helm

Reply:
updated in values sample yaml